new.js

```diff
@@ -0,0 +1,161 @@
+sum(5,10,15);
+
+//Q2
+//Create a function named colorCar
+
+function colorCar(color){
+console.log("it's a " + color + " car!");
+}
+colorCar("yellow");
+
+//Q3
+//Create an object and a function that takes the object as a parameter and prints out all of its properties and values
+const movie = {
+    name: "zohan",
+    year: 2000,
+    genre: "comedy",
+    actor: "adam sandler"
+    };
+
+    function movieInfo(object){
+        let entries = Object.entries(object);
+        for ( let [key,value] of entries){
+    console.log(key + " : " + value);
+        }  
+    }
+    movieInfo(movie);
+
+//Q4
+//Create a function named vehicleType
+function vehicleType(color,type){
+    if(type == 1){
+        console.log("a " + color + " car");
+    }else if (type ==2){
+        console.log("a " + color + " motorbike");
+    }else if (type ==3){
+        console.log("a " + color + " bicycle");
+    }
+}
+vehicleType("blue",1);
+vehicleType("green",2);
+vehicleType("red",3);
+
+//Q5
+// Can you write the following without the if statement, but with just as a single line with console.log(...);?
+// if (3 === 3) {
+//   console.log("yes");
+// } else {
+//   console.log("no");
+// }
+
+console.log((3 === 3) ? "yes" : "no");
+
+//Q6
+//Create a function called vehicle
+
+function vehicle(color,type,age){
+    if(type == 1 && age == 1){
+        console.log("a new " + color + " car");
+    }else if(type == 1 && age == 2){
+        console.log("a used " + color + " car");
+    }else if (type == 2 && age == 1){
+        console.log("a new " + color + " motorbike");
+    }else if (type == 2 && age == 2){
+        console.log("a used " + color + " motorbike");
+    }
+}
+vehicle("blue",1,1);
+vehicle("green",2,2);
+
+//Q7
+//Make a list of vehicles
+
+let listOfVehicles = [ "car","motorbike", "caravan", "bike"];
+
+//Q8
+//How do you get the third element from list?
+
+console.log(listOfVehicles[2]);
+
+//Q9
+//Change the function vehicle to use the list of question 7. 
+
+function vehicle(color,type,age){
+    if(type == 1 && age == 1){
+        console.log("a new " + color + " " + listOfVehicles[0]);
+    }else if(type == 1 && age == 2){
+        console.log("a used " + color + " " + listOfVehicles[0]);
+    }else if (type == 2 && age == 1){
+        console.log("a new " + color + " " + listOfVehicles[1]);
+    }else if (type == 2 && age == 2){
+        console.log("a used " + color + " " + listOfVehicles[1]);
+    }
+}
+vehicle("blue",1,1);
+vehicle("green",2,2);
+
+//Q10
+//Use the list of vehicles to write an advertisement. 
+//the list is already created above
+
+let adv = "Amazing hani's Garage, we service ";
+function advertisement (){
+    for( i = 0 ; i < listOfVehicles.length; i++){
+    
+      if (i == listOfVehicles.length-1){
+        adv +=  " and " + listOfVehicles[i] + ".";
+      }else if(i == listOfVehicles.length-2){
+         adv += listOfVehicles[i];
+      }
+      else { adv += listOfVehicles[i] +  ", " ;
+      }
+}console.log(adv)
+}
```

Comment:
This is fine, but it could be a bit more readable if you add more spaces, like this:

```
function advertisement (){
    for( i = 0 ; i < listOfVehicles.length; i++) {
      if (i == listOfVehicles.length-1){
        adv +=  " and " + listOfVehicles[i] + ".";
      } else if(i == listOfVehicles.length-2) {
         adv += listOfVehicles[i];
      } else { 
         adv += listOfVehicles[i] +  ", " ;
      }
   }
  console.log(adv);
}
```

Then it is also easier to see which `{` matches to the `}`